Applications/IRCClient/IRCClient.h

```diff
@@ -61,6 +61,7 @@ class IRCClient final : public Core::Object {
     void change_nick(const String&);
 
     bool is_nick_prefix(char) const;
+    bool is_channel_prefix(char) const;
```

Comment:
Neither of these actually need `this`, so they could both be `static` functions :)

Reply:
Fixed